libkbfs/mdserver_remote.go

```diff
@@ -32,21 +32,29 @@ const (
 	// MdServerDefaultPingIntervalSeconds is the default interval on which the
 	// client should contact the MD Server
 	MdServerDefaultPingIntervalSeconds = 10
+	// MdServerPingTimeout is how long to wait for a ping response
+	// before breaking the connection and trying to reconnect.
+	MdServerPingTimeout = 30 * time.Second
 )
 
 // MDServerRemote is an implementation of the MDServer interface.
 type MDServerRemote struct {
-	config       Config
-	conn         *rpc.Connection
-	client       keybase1.MetadataClient
-	log          logger.Logger
-	mdSrvAddr    string
-	authToken    *kbfscrypto.AuthToken
-	squelchRekey bool
+	config        Config
```

Comment:
It might be good to refactor this to an `mdServerRemoteConfig`, that only includes the `Config` methods we need.